osu.Server.BeatmapSubmission/Program.cs

```diff
@@ -30,6 +30,9 @@ public static void Main(string[] args)
             {
                 options.Filters.Add<InvariantExceptionFilter>();
                 options.Filters.Add<ModelStateValidationFilter>();
+
+                if (AppSettings.UserAllowList != null)
```

Comment:
It's not a huge deal at all, but due to the getter properties the parsing logic is run twice here.

Reply:
Fair point. Think it's worth switching the null check to a `is int[]` null-check-but-also-variable-declaration instead?

Reply:
can do. and maybe just hashset the whole way rather than array to ienum to hashset